src/modules/index.ts

```diff
@@ -62,6 +63,7 @@ export const registerModules: VueStorefrontModule[] = [
   PromotedOffers,
   Magento2CMS,
   // GoogleAnalytics,
+  Hotjar,
```

Comment:
Please keep it commented by default so it's not bundled while not needed

Reply:
@filrak done